internal/datasourcev2/streaming_data_source.go

```diff
@@ -161,6 +173,8 @@ func (sp *StreamProcessor) consumeStream(stream *es.Stream, closeWhenReady chan<
 
 			sp.logConnectionResult(true)
 
+			// TODO(cwaldren/mkeeler): Should this actually be true by default? It means if we receive an event
```

Comment:
I think leaving it as is would result in the same behavior we currently have, right? So it's probably fine to keep it.

We could consider setting it to false until we get a useful event. But practically I doubt it would make much of a difference.